plugin/hier.vim

```diff
@@ -64,11 +42,11 @@ function! s:Hier(clearonly)
 	for type in ['qf', 'loc']
 		for i in s:Getlist(0, type)
 			if i.bufnr == bufnr
-				let hi_group = 'QFError'
+				let hi_group = eval('g:hier_highlight_group_'.type)
 				if i.type == 'I' || i.type == 'info'
-					let hi_group = 'QFInfo'
+					let hi_group = eval('g:hier_highlight_group_'.type.'i')
 				elseif i.type == 'W' || i.type == 'warning'
-					let hi_group = 'QFWarning'
+					let hi_group = eval('g:hier_highlight_group_'.type.'w')
```

Comment:
Could you replace `eval()` with `get()` ?

See. https://twitter.com/pink_bangbi/status/810300975357591557